### DISCUSSION TITLE
how to change user or log out user?

### DISCUSSION THREAD

**Discussion by filipagh** [fief-dev/fief#152] [Q&A] [answered]
how can i change logged user 

after im logged in 1. time i want change user in same browser session (log out and show login page) how it is possible? 

looks like fief hold some session data

**Comment by frankie567** [comment, marked as answer]
Yes, after clearing your own session you need to redirect to the `/logout` page of Fief so session on Fief's side is also cleared. There is a dedicated method in the Python client to generate this URL: https://fief-dev.github.io/fief-python/fief_client.html#Fief.logout_url
